org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/utils/FilesUtilsTest.java

```diff
@@ -27,18 +28,19 @@
 	public void testFilesCachePathPreference() throws Exception {	
 		System.clearProperty(FilesUtils.LSP4XML_WORKDIR_KEY);
 		String newBasePathString = System.getProperty("user.home");
-		String newSubPathString = "New/Sub/Path";
+		String newSubPathString = Paths.get("New", "Sub", "Path").toString();
 		Path newSubPath = Paths.get(newSubPathString);
 		FilesUtils.setCachePathSetting(newBasePathString);
 		Path finalPath = FilesUtils.getDeployedPath(newSubPath);
-		assertEquals(newBasePathString + "/" + newSubPathString, finalPath.toString());
+		assertEquals(Paths.get(newBasePathString, newSubPathString).toString(), finalPath.toString());
 	}
 
 	@Test
 	public void normalizePathTest() {
-		assertEquals(System.getProperty("user.home") + "/Test/Folder", FilesUtils.normalizePath("~/Test/Folder"));
-		assertEquals("/Test/~/Folder", FilesUtils.normalizePath("/Test/~/Folder"));
-		assertEquals("~/Test/Folder", FilesUtils.normalizePath("./~/Test/Folder"));
-		assertEquals("/Folder", FilesUtils.normalizePath("/Test/../Folder"));
+		assertEquals(Paths.get(System.getProperty("user.home"), "Test", "Folder").toString(), FilesUtils.normalizePath("~/Test/Folder"));
+		assertEquals(Paths.get(separator, "Test", "~", "Folder").toString(), FilesUtils.normalizePath("/Test/~/Folder"));
```

Comment:
@fbricon This test was failing for me on Windows, an extra '\' was inserted at the beginning of the output of Paths.get(...) could you check if you have the same issue